src/EventListener/AddRateLimitHeadersListener.php

```diff
@@ -10,19 +10,8 @@
 
 class AddRateLimitHeadersListener implements EventSubscriberInterface
 {
-    private bool $displayHeaders;
-
-    public function __construct(bool $displayHeaders)
-    {
-        $this->displayHeaders = $displayHeaders;
-    }
-
     public function onKernelResponse(ResponseEvent $event): void
     {
-        if (!$this->displayHeaders) {
-            return;
-        }
-
```

Comment:
I think we should keep this feature. For some reason, some projects don't want to expose information about rate limiting in the response header.
If you want to display information in header but not in http response, you can add a second parameters like: `display_headers` and `display_response` or something like that

Reply:
@Melfo01 Don't understand why we want to rateLimit and let the person who make the request in a totally dark about when i can call again. Any tool that allows rate limiting send, at a minimum, some headers to give some informations. We can discuss about that.